src/Scrolling.md

```diff
@@ -4,14 +4,17 @@
 These registers can be accessed even during Mode 3, but modifications may not take
 effect immediately (see further below).
 
-## FF42–FF43 — SCY, SCX: Viewport Y position, X position
+## FF42–FF43 — SCY, SCX: Background viewport Y position, X position
 
-Those specify the top-left coordinates of the visible 160×144 pixel area within the
+These two registers specify the top-left coordinates of the visible 160×144 pixel area within the
 256×256 pixels BG map. Values in the range 0–255 may be used.
 
+The PPU calculates the bottom-right coordinates of the viewport with those formulas: `bottom := (SCY + 143) & 255` and `right := (SCX + 159) & 255`.
+As we can infer, in case the values are larger than 255 they will wrap around towards the top-left corner of the screen.
```

Comment:
```suggestion
The PPU calculates the bottom-right coordinates of the viewport with those formulas: `bottom := (SCY + 143) & 255` and `right := (SCX + 159) & 255`.
As we can infer, in case the values are larger than 255 they will wrap around towards the top-left corner of the tilemap.
```

...though the wording here sounds a little confusing/confused to me. May I suggest the following:

```suggestion
The coordinates of the bottom-right on-screen pixel are `Y = (SCY + 143) % 256` and `X = (SCX + 159) % 256`.
As suggested by the modulos, the tilemaps effectively "wrap around" both horizontally and vertically.
```

A diagram (something like the OAM chapter's OBJ positions, or screenshot(s?) of any VRAM viewer) may also convey the point more succinctly. What do you think?